runtime/j9vm/javanextvmi.c

```diff
@@ -219,8 +219,12 @@ JVM_IsContinuationsSupported(void)
 JNIEXPORT jboolean JNICALL
```

Comment:
you can remove the assertion for `Jenkins test sanity xLinux jdk11` and just return false

Reply:
done